.ruby-version

```diff
@@ -1 +1 @@
-2.1.5
+2.3.4
```

Comment:
I updated the rvm version to match our version as well since it was lagging pretty far behind.